templates/wechatprogram/project.config.json

```diff
@@ -43,7 +43,7 @@
     },
     "compileType": "miniprogram",
     "libVersion": "2.19.4",
-    "appid": "wx160703e6ec9ed84e",
+    "appid": "wx2c815291f1aa4d1c",
```

Comment:
Will it change every time?

Reply:
Just for testing on my device more convenient.